alf/environments/alf_wrappers.py

```diff
@@ -130,6 +130,9 @@ def __init__(self, env, duration):
         super(TimeLimit, self).__init__(env)
         self._duration = duration
         self._num_steps = None
+        assert self.batch_size is None or self.batch_size == 1, (
```

Comment:
Would be best to also support self.batch_size>1 using conditional_update or torch.where to reset the step_type.

Reply:
I have thought about this, but decided not to so it right now since the reset behavior for batch_size > 1 is not clearly defined.